### PR TITLE
Create bike_md.yml

### PR DESCRIPTION
**Pull request by Josiah-DuBose**
Hello, 
BikeMD is a new open source project. There is lots of work to be done so I'll be adding a lot more issues soon!

### CODE REVIEW

File: bike_md.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+desc: *A community for motorcyle technicians and owners. Post problems and solutions, get help with hard to solve issues.*
+site: *https://github.com/bike-md/bike_md*
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- *python*
+- *javascript*
+- *django*
+upforgrabs:
+  name: *up for grabs*
+  link: *https://github.com/bike-md/bike_md/issues*
```
Comment (shiftkey) [inline bike_md.yml]: Make this `https://github.com/bike-md/bike_md/issues/labels/up%20for%20grabs`

File: bike_md.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: *BikeMD*
```
Comment (shiftkey) [inline bike_md.yml]: No need to wrap these values in `*`

**Review by shiftkey — CHANGES_REQUESTED** [review]
Looking good, just a couple of suggestions to clean this up.

**Comment by Josiah-DuBose** [comment]
Done!

**Comment by shiftkey** [comment]
@Josiah-DuBose thanks!

**Pull request merged**